hi-ml/src/health_ml/experiment_config.py

```diff
@@ -87,10 +95,23 @@ class ExperimentConfig(param.Parameterized):
         doc="The maximum runtime that is allowed for this job in AzureML. This is given as a floating"
         "point number with a string suffix s, m, h, d for seconds, minutes, hours, day. Examples: '3.5h', '2d'",
     )
-    mode: str = param.ClassSelector(
+    mode: RunnerMode = param.ClassSelector(
         class_=RunnerMode,
         default=RunnerMode.TRAIN,
         doc=f"The mode to run the experiment in. Can be one of '{RunnerMode.TRAIN}' (training and evaluation on the "
         f"test set), or '{RunnerMode.EVAL_FULL}' for evaluation on the full dataset specified by the "
         "'get_eval_data_module' method of the container.",
     )
+    log_level: Optional[RunnerMode] = param.ClassSelector(
+        class_=LogLevel,
+        default=None,
+        doc=f"The log level to use. Can be one of '{LogLevel.ERROR}', '{LogLevel.WARNING}', '{LogLevel.INFO}', "
```

Comment:
replace log levels in docu by a list, note that if you add a loglevel in teh future you'll have to remember updating the docu as well, using a list is more maintainable
```suggestion
        doc=f"The log level to use. Can be one of {list(map(str, LogLevel))} "
```